_posts/2017-06-21-next-destination-zurich.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Next stop: Zürich!"
+socialImageSrc: 'A BIG FAT TODO RIGHT HERE'
+authors:
+  - andrew
+  - gandro
+---
+
+It's time to begin another adventure! After previous fun in Berlin and Kyiv, we're getting ready for Zürich, Switzerland! We hope you'll join us on **September 30th and October 1st, 2017** at [focusTerra](https://goo.gl/maps/cPY6GJoF8Hm) near ETH.
```

Comment:
Hm.. so should careful how we want to call the venue, since people might get confused otherwise. Let me explain ;)

The reserved rooms are in a building named "NO", which part of the ETH Zurich Zentrum campus. Within this building is a permanent exhibition called "focusTerra". 

While it is generally a good idea to tell people that they should follow the "focusTerra" signs to find the venue, the venue itself is called NO and the focusTerra people might want to be associated with us (since they are just an exhibition who happens to be in the same building as us, but otherwise there is no relation). 

This is the "homepage" of the building itself - unfortunately, Google Maps does not know about it: https://www.erdw.ethz.ch/en/department/location.html#no